crates/bevy_render/src/primitives/mod.rs

```diff
@@ -123,9 +154,24 @@ impl HalfSpace {
     }
 }
 
-/// A frustum made up of the 6 defining half spaces.
-/// Half spaces are ordered left, right, top, bottom, near, far.
-/// The normal vectors of the half spaces point towards the interior of the frustum.
+/// A frustum defined as the intersection of 6 [`HalfSpace`].
+/// Usually a six sided polyhedron with sides inside the bisecting planes of the half-spaces.
```

Comment:
It can be a cuboid for orthogonal projection, or I guess technically it could have less than six sides or extend to infinity.

But I did say "Usually" and a Cuboid is a special case of truncated pyramid I guess.

According to wikipedia, the real name of the truncated pyramid is either "Quadrilateral frustum" (which doesn't help much here) or "apex-truncated square pyramid" which I think works? Are you okay with the last one?